nextflow_schema.json

```diff
@@ -553,6 +553,7 @@
                 },
                 "cluster_options": {
```

Comment:
I'm still a bit nervous about this switch - this param isn't used anywhere currently that I'm aware of, nf-core/configs is still all camelCase:

https://github.com/nf-core/configs/blob/26b5904df15ab28edcd4447cfe8e290e7da85a4f/conf/uppmax.config#L14

But you said that the schema linter won't allow the pipeline to launch if it's camelCase or something?

Phil

Reply:
So it looks like I changed this to snake_case in the template a while back and then reverted it (after a review from you actually! 😉 ) https://github.com/nf-core/tools/pull/438

I wonder if a better approach would be to simply get the schema validator to ignore it instead, and omit it from the schema...

https://github.com/nf-core/methylseq/blob/7a148a33fcf304c7ffa5bc70716867447f4e0b0e/nextflow.config#L79

Reply:
All removed 👍🏻